src/SB/Game/zCamera.cpp

```diff
@@ -890,7 +897,7 @@ void zCameraMinTargetHeightSet(F32 min_height)
 
 void zCameraMinTargetHeightClear()
 {
-    zcam_mintgtheight = zCamera_f_n1_0_e38;
+    zcam_mintgtheight = -1.0e38;
```

Comment:
Missed an f suffix here, would be good to add to avoid accidental cast to float and to keep the style consistent.